reactRouter/Index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import List from './list.jsx';
+import request from 'sync-request';
+import AddList from './addList.jsx';
+import ClientList from './clientList.jsx';
+
+import { Router, Route, Link, IndexRoute, hashHistory, browserHistory, DefaultRoute } from 'react-router'
+class Index extends Component {
+  render () {
+    return (
+      <Router history={hashHistory}>
```

Comment:
@VARTIKASINGH1234 Why you've used the 'hashHistory' not 'browserHistory'? Please clarify the point?